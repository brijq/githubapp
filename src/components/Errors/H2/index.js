import React from 'react'
import PropTypes from 'prop-types'

class H2 extends React.Component {
  render() {
    return(
      <h2 className="text-center">
        <span className="fa fa-remove"/>&nbsp;{this.props.message}
      </h2>
    )
  }
}

H2.propTypes = {
  message: PropTypes.string.isRequired
};

export default H2
