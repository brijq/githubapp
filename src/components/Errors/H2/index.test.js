import React from 'react'
import H2 from './index'
import renderer from 'react-test-renderer'

test("renders correctly", () => {
  const h2 = renderer.create(<H2
    message="hello"
  />).toJSON();
  expect(h2).toMatchSnapshot();
});
