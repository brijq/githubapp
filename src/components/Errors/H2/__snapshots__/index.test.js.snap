exports[`test renders correctly 1`] = `
<h2
  className="text-center">
  <span
    className="fa fa-remove" />
  Â 
  hello
</h2>
`;
